src/pages/components/testimonial/index.js

```diff
@@ -53,19 +58,27 @@ const Testimonial = () => {
           }}
           {...settings}
         >
-          {testimonials.map((item, index) => (
-            <TestimonialCard key={index} item={item} />
-          ))}
+          {testimonials.map(
+            (item, index) =>
+              name !== item.author && (
+                <Link
+                  to={`/testimonial-content/${item.author}`}
```

Comment:
```suggestion
                  to={`/swipet-journeys/${item.author}`}
```
Can we change the path to /swipet-journeys/ ?